### PR TITLE
Add sprint report for Sprint 21

### PR DESCRIPTION
**Pull request by DavidKang**

### CODE REVIEW

File: _posts/development/2017-08-07-sprint-report-7.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+<p>
+  Here are the results the OBS frontend team has achieved in the last two weeks (2017-07-24 to 2017-08-04).
+</p>
+
+<h3>Features</h3>
+<h4>Kiwi Editor</h4>
+
+<p>
+  In this sprint, we did a lot of work related to KIWI Editor, we have added more functionality to the Kiwi image editor,
+  now the package list can be edited together with the repositories. We also improve the KIWI editor interface,
```
Comment (bgeuken) [inline _posts/development/2017-08-07-sprint-report-7.md]: We also improved

File: _posts/development/2017-08-07-sprint-report-7.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+<p>
+  Here are the results the OBS frontend team has achieved in the last two weeks (2017-07-24 to 2017-08-04).
+</p>
+
+<h3>Features</h3>
+<h4>Kiwi Editor</h4>
+
+<p>
+  In this sprint, we did a lot of work related to KIWI Editor, we have added more functionality to the Kiwi image editor,
```
Comment (bgeuken) [inline _posts/development/2017-08-07-sprint-report-7.md]: I would replace the two commas here. "In this sprint, we did a lot of work related to KIWI Editor. We have added ... to the kiwi image editor. Now ..."

File: _posts/development/2017-08-07-sprint-report-7.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+  <img src="/images/posts/sprint_21_kiwi_show.png" alt= 'KIWI show view'>
+  <img src="/images/posts/sprint_20_kiwi_repository_actions_button.png" alt= 'KIWI repository actions button'>
+  <img src="/images/posts/sprint_20_kiwi_repository_form.png" alt= 'KIWI repository form'>
+</p>
+
+<h3>Testing</h3>
+<h4>PacKInfo</h4>
+
+<p>
+  We still working in your coverage, having a sane coverage is having a healthy application :).
```
Comment (bgeuken) [inline _posts/development/2017-08-07-sprint-report-7.md]: We still work or We are still working

Comment (bgeuken) [inline _posts/development/2017-08-07-sprint-report-7.md]: And I think it's working on. But I would double ckeck with @evanrolfe  ^^

File: _posts/development/2017-08-07-sprint-report-7.md [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+  If you're interested look into this PR: <a href="https://github.com/openSUSE/open-build-service/pull/3469">#3469</a>
```
Comment (bgeuken) [inline _posts/development/2017-08-07-sprint-report-7.md]: have a look sounds nicer IMO, more inviting.

File: _posts/development/2017-08-07-sprint-report-7.md [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+</p>
+
+<h3>Bugs</h3>
+<h4>Killing Deadlocks</h4>
+
+<p>
+  We also deal with deadlocks :(. It turns out that touching Project on all changes of Package produce a SQL deadlocks because we have very long running transactions for `Project`.
```
Comment (bgeuken) [inline _posts/development/2017-08-07-sprint-report-7.md]: ``` dealt ```

File: _posts/development/2017-08-07-sprint-report-7.md [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+<p>
+  In this sprint, we did a lot of work related to KIWI Editor, we have added more functionality to the Kiwi image editor,
+  now the package list can be edited together with the repositories. We also improve the KIWI editor interface,
+  mimicking SUSE Studio, so now it's easier to add, edit and delete repositories and packages.
+</p>
+<p>
+  Here is how it currently looks like:
+  <img src="/images/posts/sprint_21_kiwi_show.png" alt= 'KIWI show view'>
+  <img src="/images/posts/sprint_20_kiwi_repository_actions_button.png" alt= 'KIWI repository actions button'>
+  <img src="/images/posts/sprint_20_kiwi_repository_form.png" alt= 'KIWI repository form'>
```
Comment (bgeuken) [inline _posts/development/2017-08-07-sprint-report-7.md]: The two last image are not shown in the ui. You probably just have to fix the naming (sprint_20 -> sprint_21).

File: _posts/development/2017-08-07-sprint-report-7.md [hunk 7]
```diff
@@ -0,0 +1,41 @@
...
+</p>
+
+<h3>Features</h3>
+<h4>Kiwi Editor</h4>
+
+<p>
+  In this sprint, we did a lot of work related to KIWI Editor, we have added more functionality to the Kiwi image editor,
+  now the package list can be edited together with the repositories. We also improve the KIWI editor interface,
+  mimicking SUSE Studio, so now it's easier to add, edit and delete repositories and packages.
+</p>
```
Comment (bgeuken) [inline _posts/development/2017-08-07-sprint-report-7.md]: Not super-important, but you could also mention the follow up card (https://trello.com/c/U8tNF6K6/385-improve-current-kiwi-image-editor) we created during the review.

**Comment by bgeuken** [comment]
Looks good:+1:

Once you fixed the few typos and updated the image links, we can merge this.

**Comment by DavidKang** [comment]
@bgeuken, could you re-review it :smile_cat:

**Comment by bgeuken** [comment]
Great!

Let's ship it

**Pull request merged**